### PR TITLE
DLPJTS-97 New example draws boxes around each character.

### PR DESCRIPTION
**Pull request by datalogics-kam**
**Please do [pdf-java-toolkit#978](https://octocat.dlogics.com/datalogics/pdf-java-toolkit/pull/978) first (this PR may not build correctly until then either).**
#### Changes in this pull request
- Demonstrates the easier-to-use method of getting character quads in PDFJT 4.2.
- Requires PDFJT 4.2 and Talkeetna 6.
- This sample and its test are excluded from building with older versions.
#### Checklist for approving this pull request

(PR creator amend this with more conditions if necessary)
- [x] There are **unit tests** for new/changed code, or a good explanation why this was not possible.
- [x] All **CI builders** have indicated success (Give them a few minutes to notice the pull request.)
- [x] The **Pull request title** has the JIRA issue numbers separated by spaces (if any), a space, and then a short, but descriptive summary.
- [x] **Commit messages** are well formed: [A note about Git commit messages](http://www.tpope.net/node/106)
- [x] New public packages, classes, and methods are **documented**. (Strongly consider documenting private classes and methods.)
#### Blockers

Add a checkbox for yourself with your **&#64;name** to this list if you are holding this PR open for review. PR Shepherd, please hold off merging this PR until these are all checked:
- [x] _&lt;add your name here with an **@**>_

### CODE REVIEW

**Comment by datalogics-devauto** [comment]
Build succeeded.

**Comment by datalogics-devauto** [comment]
Build failed.

File: pom.xml [hunk 1]
```diff
@@ -151,12 +151,18 @@
...
+          <testExcludes>
+            <testExclude>com/datalogics/pdf/samples/extraction/TextCharacterBoxesTest.java</testExclude>
+          </testExcludes>
         </configuration>
       </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-javadoc-plugin</artifactId>
-        <version>2.10.1</version>
+        <version>2.10.4</version>
```
Comment (datalogics-coreyy) [inline pom.xml:165]: Can you explain this bump here from 1 to 4?

Comment (datalogics-kam) [reply pom.xml:165]: A lot of times when I have trouble getting things to work, I update to the latest plugin along the way, just to make sure.

**Comment by datalogics-coreyy** [comment]
I'm not sure what the purpose of TextCharacterBoxesTest.pdf.page0.txt and TextCharacterBoxesTest.pdf.page1.txt serves.

**Comment by datalogics-coreyy** [comment]
All looks good here and tests fine on my machine. I'd just ask @datalogics-robl to spot check the change in pom.xml. Are we concerned about the default check failing (probably just because the other pr wasn't merged yet)?

**Comment by datalogics-devauto** [comment]
Build succeeded.

**Comment by datalogics-devauto** [comment]
Build failed.

**Comment by datalogics-devauto** [comment]
Build succeeded.

**Comment by datalogics-devauto** [comment]
Build succeeded.

**Comment by datalogics-coreyy** [comment]
I'll retest shortly here and if all goes well, well you know the rest

**Pull request merged**